python/sidecar/typing.py

```diff
@@ -201,6 +201,14 @@ class SearchResponse(RefStudioModel):
     results: list[S2SearchResult] 
 
 
+class UpdateSettingsRequest(RefStudioModel):
+    settings: dict[str, Any]
+
+
+class SettingsResponse(RefStudioModel):
+    settings: dict[str, Any]
+
+
```

Comment:
We know the structure of the settings. Can't we have a strongly typed definition?